examples/plot_compare_reduction.py

```diff
@@ -42,14 +43,16 @@
         'classify__C': C_OPTIONS
     },
     {
-        'reduce_dim': [SelectKBest(chi2)],
+        'reduce_dim': [SelectKBest(chi2), SelectKBest(info_gain),
```

Comment:
I've added an example comparing CHI^2, MI, F, IG and IGR on the 20 newsgroups data, plotting accuracy as function of the amount of selected features (see the figure). I've removed IG and IGR from other examples, except `plot_compare_reduction.py`, where I left IG - I think it would be good to illustrate it on at least  one other dataset. 
![figure_1](https://cloud.githubusercontent.com/assets/1921642/21765680/586d5aaa-d662-11e6-875d-ca2efe2330c2.png)